Dockerfile

```diff
@@ -48,6 +48,7 @@ ARG AUTH_URL
 ARG POSTS_URL
 ARG WEB_URL
 ARG SEARCH_URL
+ARG FEED_DISCOVERY_URL
```

Comment:
You need this up in the top-level layer too (around line 21)

Reply:
Good catch, fixed!